src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableActorContext.cs

```diff
@@ -82,25 +84,42 @@ object IDurableActorContext.GetOperationContent(Type argumentType)
             return this.CurrentOperation.GetContent(argumentType);
         }
 
-        IStateView<TState> IDurableActorContext.GetState<TState>(Formatting formatting, JsonSerializerSettings settings)
+        T IDurableActorContext.GetState<T>()
         {
             this.ThrowIfInvalidAccess();
-            if (this.CurrentStateView != null)
+
+            if (!this.StateWasAccessed)
+            {
+                var result = (this.State.ActorState == null)
+                    ? default(T)
+                    : JsonConvert.DeserializeObject<T>(this.State.ActorState);
```

Comment:
I believe we want to use `MessagePayloadDataConverter.Default` here.

Reply:
yes.